taxi/migrations/0002_alter_driver_license_number.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.1 on 2024-10-28 20:46
+
+from django.db import migrations, models
+import taxi.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("taxi", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="driver",
+            name="license_number",
+            field=models.CharField(
+                max_length=255,
+                unique=True,
+                validators=[taxi.models.clean_license_number],
```

Comment:
Ensure that the `clean_license_number` function is defined in `taxi.models`. If it is not defined, this will cause an error during migration execution.